Kickstarter-iOS/Views/Controllers/RewardsCollectionViewController.swift

```diff
@@ -307,13 +324,31 @@ final class RewardsCollectionViewController: UICollectionViewController {
     self.navigationController?.pushViewController(pledgeViewController, animated: true)
   }
 
-  // MARK: - Public Functions
+  // MARK: - Actions
 
   @objc func closeButtonTapped() {
     self.navigationController?.dismiss(animated: true)
   }
 }
 
+// MARK: - Functions
+
+private func newShadowImage(with color: UIColor) -> UIImage? {
```

Comment:
Could this be an extension on `UIImage`?

```
let color = UIColor.ksr_grey_200
UIImage.image(with: color)
```

Reply:
Could be... wasn't sure if we'd need it anywhere else?